components/Feedback.js

```diff
@@ -0,0 +1,83 @@
+import { useState } from "react";
+import { useRouter } from "next/router";
+
+export default function Feedback(className = '') {
+    const [Submit, setSubmit] = useState(false);
+    const [feedback, setFeedback] = useState('')
+    const date_stamp = new Date()
+    const time_stamp = date_stamp.getDate().toString() + '/' + (date_stamp.getMonth()+1).toString() + '/' + date_stamp.getFullYear().toString() + ' ' + date_stamp.getHours().toString() + ':' + date_stamp.getMinutes().toString() + ':' + date_stamp.getSeconds().toString();
+    const { asPath, pathname } = useRouter();
+    async function handleSubmit(e) {
+        e.preventDefault();
+        const data = {
+            title: 'Feedback on ' + asPath + ' - ' + time_stamp,
+            feedback: feedback
+        }
+
+        const response = await fetch("/.netlify/functions/handler", {
```

Comment:
You don't need to wait for it:

```suggestion
        fetch("/.netlify/functions/handler", {
```

Also please implement some error handling by `.catch()` method. We should inform user that something went wrong (for example problem with GitHub API) and user should know about it and try later.

Reply:
In order to tell the user that something went we need to design a similar kind of stuff but showing the error. Will implement this in the upcoming commit.